test/web-platform-tests/to-upstream/html/semantics/forms/the-input-element/checkbox-click-events.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Checkbox click events</title>
+<link rel="author" title="jeffcarp" href="mailto:gcarpenterv@gmail.com">
```

Comment:
It's certainly optional, but I like filling it out personally so that no matter which repo these get copied into, you still get to say "I did that!" :)